tsdat/io/retrievers/storage_retriever.py

```diff
@@ -35,6 +35,7 @@ class TransParameters(BaseModel):
 
     parameters: Optional[TransParameters] = None
 
+    # TODO: `input_data_hook` is not included in docstring.
```

Comment:
+1

Reply:
I can spend time reading over the code to write up a decent docstring segment for `input_data_hook`, but I suspect it'd be easier for you or @jmcvey3 to just write up one here instead.

Reply:
yeah, I'm fine if this is left as a todo and I can get around to it later